pkg/sql/sem/tree/datum.go

```diff
@@ -2980,6 +2980,8 @@ func AsJSON(d Datum, loc *time.Location) (json.JSON, error) {
 		return json.FromString(string(*t)), nil
 	case *DCollatedString:
 		return json.FromString(t.Contents), nil
+	case *DEnum:
```

Comment:
i've missed this several times now. we should probably have a test that goes through all known datum types and generates a json and make sure this exists, but if you're not bothered to do that atm, file an issue on me?

Reply:
I'll open an issue for it